fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -350,6 +350,15 @@ public boolean isAdmin() {
     return true; // TODO(ttomsu): Chosen by fair dice roll. Guaranteed to be random.
   }
 
+
+  public boolean isAdmin(Authentication authentication) {
+    if (!fiatStatus.isEnabled()) {
+      return true;
+    }
+
+    return getPermission(getUsername(authentication)).isAdmin();
```

Comment:
I think it's possible that `getPermission` could return null (or raise an exception) ... worth a null check and possibly a try/catch that would return false rather than propagate the exception.